Applications/Spire/Include/Spire/Ui/QuantityBox.hpp

```diff
@@ -41,6 +41,9 @@ namespace Spire {
       QuantityBox(std::shared_ptr<OptionalQuantityModel> model,
         QHash<Qt::KeyboardModifier, Nexus::Quantity> modifiers,
         QWidget* parent = nullptr);
+
+    private:
+      struct UnsignedQuantityModel;
```

Comment:
If this is only used by the QuantityBox source file, can it just be declared and defined in an anonymous namespace in the QuantityBox.cpp file?

Reply:
Because it is only used by the QuantityBox, it is declared the nested class of the QuantityBox, which works like DecimalBox::DecimalToTextModel.

Reply:
Okay, I see. I didn't notice there was a precedent.